waiter/src/waiter/scheduler/shell_scheduler.clj

```diff
@@ -8,7 +8,7 @@
 ;;       The copyright notice above does not evidence any
 ;;       actual or intended publication of such source code.
 ;;
-(ns waiter.shell-scheduler
+(ns waiter.scheduler.shell-scheduler
```

Comment:
Now that we have `scheduler` in the namespace, the `-scheduler` in `shell-scheduler` seems redundant.  What do you think?

Reply:
I'm ok renaming it to `shell.clj`. Or do you prefer a different name?

Reply:
Renamed to `shell.clj`